econml/utilities.py

```diff
@@ -1391,6 +1393,91 @@ def predict(self, X):
         return self._clf.predict_proba(X)[:, 1:]
 
 
+class _TransformerWrapper:
+    """Wrapper that takes a featurizer as input and adds jacobian calculation functionality"""
+
+    def __init__(self, featurizer):
+        pass
+        self.featurizer = featurizer
+
+    def fit(self, X):
+        return self.featurizer.fit(X)
+
+    def transform(self, X):
+        return self.featurizer.transform(X)
+
+    def fit_transform(self, X):
+        return self.featurizer.fit_transform(X)
+
+    def get_feature_names(self, X):
+        if hasattr(self.featurizer, 'get_feature_names'):
+            return self.featurizer.get_feature_names(X)
+
+        return []
+
+    def jac(self, X):
+        if hasattr(self.featurizer, 'jac'):
+            return self.featurizer.jac(X)
+        elif (isinstance(self.featurizer, PolynomialFeatures)):
+            self.featurizer.fit(X)
+            powers = self.featurizer.powers_
+            result = np.zeros(X.shape + (self.featurizer.n_output_features_,))
+            for i in range(X.shape[1]):
+                p = powers.copy()
+                c = powers[:, i]
+                p[:, i] -= 1
+                M = np.float_power(X[:, np.newaxis, :], p[np.newaxis, :, :])
+                result[:, i, :] = c[np.newaxis, :] * np.prod(M, axis=-1)
+            return result
+
+        else:
+
+            squeeze = []
+
+            n = X.shape[0]
+
+            d_t = X.shape[-1] if ndim(X) > 1 else 1
+
+            X_out = self.transform(X)
+            d_f_t = X_out.shape[-1] if ndim(X_out) > 1 else 1
+
+            epsilon = 0.001
```

Comment:
Have you considered making this a keyword-only argument?

Maybe there's no point because there's no good way to pass it into the callers of this method, but give it some thought.